features/step_definitions/wait_steps.rb

```diff
@@ -67,13 +67,13 @@
 Then(/^an exception is raised when I wait for a section that won't appear$/) do
   expect { @test_site.section_experiments.parent_section.wait_for_slow_section_element(1) }
     .to raise_error(SitePrism::TimeOutWaitingForExistenceError)
-    .with_message('Timed out after 1s waiting for Parent#slow_section_element')
+    .with_message('Timed out after waiting for existence.')
```

Comment:
Is there a reason we've removed the info? I know there was a discussion but I half missed it (Sorry :( )

Reply:
Several reasons, but in short it seemed like the pay-off between maintenance burden and complexity vs. benefit wasn't very high.
1) It looked nice, but wasn't actually all that useful.  It didn't really tell users anything they didn't really already know.  If one got a TimeoutWaitingForExistenceError on a line of code that typically looks something like `@page.wait_for_total_price(10)` it's a trivial problem to determine for what and for how long one might have been waiting for.
2) It would make all of this logic significantly more complex because the waiter would need to have details of the calling method internals to properly call the error (See everything that's passed to `raise_wait_for_if_failed` for details) 
3) It was at the time not available in any release yet.
4) It resulted in the addition of several public methods that were not listed in the README.
5) It was, so far, inconsistent because only a couple of errors were implemented this way.

Reply:
I agree with points 3/4/5. I think the question around 1/2 was is it worth it.

With regards to error messages, I want to completely refactor them (Breaking change), and it seems as though on the whole people aren't keen on providing the public api messages. So I think I'll resist the urge to fight back and remove them from our bindings.